index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title> Leslie Portfolio project</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body class="body">
+    <header class="header">
+        <a class="logo" href="">John Doe</a>
+        <nav class="navigate">
+        <a href=""><img src="images/Union.png" alt="" /></a>
+      </nav>
+    </header>
+    <section>
+        <div class="heading-section">
+                <h1 class="heading">Hey There <br />I’m Leslie</h1>
+                <h2 class="describe">I am a Software Developer</h2> 
+            <p class="headline">
+                I can help you build a product , feature or website Look through some of
+                my work and experience! If you like what you see and have a project you
+                need coded, don’t hestiate to contact me.</p>
+                <ul class="social">
+                    <li class="social-icons">
+                        <a href="https://mobile.twitter.com/Leslie"><img src="images/twitter.png" alt=""/></a> 
+                    </li>
+                    <li class="social-icons">
+                        <a href="https://www.linkedin.com/in/LeslieAine"><img src="images/linkedin.png" alt=""/></a>
+                     </li>
+                     <li class="social-icons">
+                        <a href="https://github.com/LeslieAine"><img src="images/github.png" alt=""/></a> 
+                    </li>
+                    <li class="social-icons">
+                        <a href="#"><img src="images/medium.png" alt=""/></a> 
+                    </li>
+                    <li class="social-icons">
+                        <a href="#"><img src="images/Angelist.png" alt=""/></a> 
+                    </li>
+                    </ul>
+    </section>
+    
+      
+        <img class="background"src="images/Group.png" alt="">
```

Comment:
- [ ]  As requested by the previous reviewer, kindly make sure to use the CSS background property to insert the background image on your page. It doesn't look professional to add a background image by using HTML. I highly recommend you to use CSS to insert your background image. :wink: